Responsive navigation bar

saabi opened the pull request:
Closes #5

saabi committed:
Initial work on responsive nav bar

mindrones commented on src/routes/_layout.svelte:
```diff
@@ -27,5 +33,11 @@
+
+	header.mobile {
+		border-top: 1px solid var(--color-main-lighter);
+		border-bottom: none;
```
I'd move this block under the `header` block

saabi committed:
Various fixes

Renamed Hamburger component
Nav bar to bottom
Fixed Scrolling
Added segment to nav bar

saabi committed:
Nav menu occupies whole screen on mobile and menu icon is now SVG

saabi committed:
Renamed hamburger, added X icon and adjusted Nav menu height

mindrones commented on src/node_modules/app/components/Nav.svelte:
```diff
@@ -1,64 +1,94 @@
 		display: flex;
 		align-items: center;
 		justify-content: space-between;
+		z-index: 1000;
```
if we really have to use `z-index` I'd prefer having it as a CSS var (say `--z-1000: 1000;`) in `global.css` so that we keep a register of where it's used

saabi committed:
Adjusted layout

Moved menu button to the right in mobile
Adjusted font size for links in mobile
Adjusted spacing around items in menu

saabi committed:
Filtering out selected route in mobile nav menu

saabi committed:
Menu divs  spaced evenly

mindrones commented on src/node_modules/app/components/Nav.svelte:
```diff
@@ -1,64 +1,102 @@
 <script>
+	import {capitalize} from '@svizzle/utils';
+
 	import ExternalLink from '@svizzle/ui/src/ExternalLink.svelte';
 	import IconDownload from '@svizzle/ui/src/icons/IconDownload.svelte';
+	import IconMenu from '@svizzle/ui/src/icons/IconMenu.svelte';
+	import IconX from '@svizzle/ui/src/icons/IconX.svelte';
+	import IconMessageSquare from '@svizzle/ui/src/icons/IconMessageSquare.svelte';
```
minor, I'd sort by package

mindrones commented on src/node_modules/app/components/Nav.svelte:
```diff
@@ -1,64 +1,102 @@
 <script>
+	import {capitalize} from '@svizzle/utils';
+
```
extra blank line to be removed

mindrones commented on src/node_modules/app/components/Nav.svelte:
```diff
@@ -1,64 +1,102 @@
+
+	function handleClick () {
+		isMenuOpen = !isMenuOpen;
+		console.log(isMenuOpen);
```
leftover

mindrones reviewed (approved):
Needs some minor edit, below, but great work! 👍

mindrones commented on src/node_modules/app/components/Nav.svelte:
```diff
@@ -1,64 +1,102 @@
 
 	import {version} from 'app/utils/version';
 
 	const changelogUrl = 'https://github.com/nestauk/beis-indicators/blob/staging/CHANGELOG.md'; // FIXME
```
```suggestion
	const changelogUrl = 'https://github.com/nestauk/eurito_indicators_ui/blob/staging/CHANGELOG.md';
```

saabi replied:
Good catch!

mindrones reviewed (approved):
I forgot, we need to update the change log

saabi committed:
Now using Svizzle Icons

saabi committed:
Added icon to Feedback link

saabi committed:
Header is now centered to the screen on mobile

saabi committed:
Added definition for Chrome browser

saabi committed:
Added comments to browser specific props

saabi committed:
Fixed comments

saabi committed:
Review comments
